params/config.go

```diff
@@ -515,30 +437,30 @@ type ChainConfig struct {
 	MuirGlacierBlock    *big.Int `json:"muirGlacierBlock,omitempty"`    // Eip-2384 (bomb delay) switch block (nil = no fork, 0 = already activated)
 
 	// Avalanche Network Upgrades
-	ApricotPhase1BlockTimestamp *uint64 `json:"apricotPhase1BlockTimestamp,omitempty"` // Apricot Phase 1 Block Timestamp (nil = no fork, 0 = already activated)
+	ApricotPhase1Time *uint64 `json:"apricotPhase1Time,omitempty"` // Apricot Phase 1 Block Timestamp (nil = no fork, 0 = already activated)
```

Comment:
This is the only bit that concerns me: presumably there will be JSON somewhere that is broken by this? I see that you updated some tests, but what needs to change in production?

Reply:
This is mostly fine since we have never used them from a JSON file. However it's concerning that we might have stored this as a part of genesis and we do check them [here](https://github.com/ava-labs/coreth/blob/master/params/config.go#L836). If I'm not mistaken this might be broken since the storedcfg will have nil timestamps.

Reply:
what is the best way to test this? is running a node from scratch sufficient?
should we have some node then apply this patch then restart it?

Reply:
yes we should have a node already stored the genesis to test this out.